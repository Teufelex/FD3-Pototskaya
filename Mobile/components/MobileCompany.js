import React from 'react';
import PropTypes from 'prop-types';
import MobileClient from './MobileClient';
import { companyEvents } from './events';

import './MobileCompany.css';

class MobileCompany extends React.PureComponent {

  static propTypes = {
    clients:PropTypes.arrayOf(
      PropTypes.shape({
        id: PropTypes.number.isRequired,
        fam: PropTypes.string.isRequired,
        im: PropTypes.string.isRequired,
        otch: PropTypes.string.isRequired,
        balance: PropTypes.number.isRequired,
      })
    ),
  };

  state = {
    initialClientsList: this.props.clients,
    workMode: 1, //1-Velcom, 2-MTC
    clients: this.props.clients,
  };

  switchToMTC = () => {
    this.setState({workMode: 2});
  }

  switchToVelcom = () => {
    this.setState({workMode: 1});
  }

  componentDidMount = () => {
    //voteEvents.addListener('EAnswerClicked',this.answerSelected);
    //voteEvents.addListener('EFreeAnswerTextChanged',this.freeAnswerTextChanged);
  };

  componentWillUnmount = () => {
    //voteEvents.removeListener('EAnswerClicked',this.answerSelected);
    //voteEvents.removeListener('EFreeAnswerTextChanged',this.freeAnswerTextChanged);
  };
  
  render() {
    console.log("MobileCompany render");

    let clients = [];
    this.state.clients.forEach(b => clients.push(<MobileClient user = {b} key = {b.id} event = {this.events}/>));

    return (
      <div className= 'MobileCompany'>
        <div className = 'MobileCompany__controlButtonsWrap'>
          <div>
            <input type = "button" value = "МТС" onClick = {this.switchToMTC}/>
            <input type = "button" value = "Velcom" onClick = {this.switchToVelcom}/>
          </div>
          <div>Компания: {(this.state.workMode === 1) ? "Velcom" : "МТС"}</div>
        </div>
        <div className = "MobileCompany__buttonsWrap">
          <input type = "button" value = "Все"/>
          <input type = "button" value = "Активный"/>
          <input type = "button" value = "Заблокированные"/>
        </div>
        <div className = "MobileCompany__clientsWrap">
          <div className = "clientsWrap__header">
            <span className = "header__cell">Фамилия</span>
            <span className = "header__cell">Имя</span>
            <span className = "header__cell">Отчество</span>
            <span className = "header__cell">Баланс</span>
            <span className = "header__cell">Статус</span>
            <span className = "header__cell">Редактировать</span>
            <span className = "header__cell">Удалить</span>
          </div>
          {clients}
        </div>
        <input type = "button" value = "Добавить клиента"/>
      </div>
    );
  }
}

export default MobileCompany;
