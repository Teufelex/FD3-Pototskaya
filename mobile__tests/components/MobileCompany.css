.MobileCompany__buttonsWrap {
  border-top: 1px solid rgb(201, 201, 201);
  border-bottom: 1px solid rgb(201, 201, 201);
  padding: 10px;
}

.MobileCompany__clientsWrap > div {
  border-top: 1px solid rgb(201, 201, 201);
  padding: 0 10px;
}

.MobileCompany__clientsWrap .clientsWrap__header {
  display: flex;
}

.clientsWrap__header .header__cell  {
  display: block;
  width: 14vw;
  padding: 15px 5px;
}