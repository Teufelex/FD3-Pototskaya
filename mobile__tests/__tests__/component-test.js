import React from 'react';
import renderer from 'react-test-renderer';
import MobileCompany from '../components/MobileCompany';

let clientsArr=[ 
  {id:101, fam:"Иванов", im:"Иван", otch:"Иванович", balance:200}, 
  {id:105, fam:"Сидоров", im:"Сидор", otch:"Сидорович", balance:250}, 
  {id:110, fam:"Петров", im:"Пётр", otch:"Петрович", balance:180},
  {id:120, fam:"Григорьев", im:"Григорий", otch:"Григорьевич", balance:220},
];

test('работа MobileCompany', () => {

  const component = renderer.create(
    <MobileCompany clients = {clientsArr}/>
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  const buttonElem = component.root.find( el => el.type==='input' && el.props.value === "Добавить клиента"); 

  buttonElem.props.onClick();

  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  buttonElem.props.onClick();

  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  const buttonElemAct = component.root.find( el => el.type==='input' && el.props.value === "Активный"); 

  buttonElemAct.props.onClick();

  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  const buttonElemBlock = component.root.find( el => el.type==='input' && el.props.value === "Заблокированные"); 

  buttonElemBlock.props.onClick();

  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  const buttonElemAll = component.root.find( el => el.type==='input' && el.props.value === "Все"); 

  buttonElemAll.props.onClick();

  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
});
