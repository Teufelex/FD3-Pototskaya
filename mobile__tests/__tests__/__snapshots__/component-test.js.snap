// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа MobileCompany 1`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Иванов
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иван
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иванович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        200
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Сидоров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидор
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидорович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        250
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Петров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Пётр
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Петрович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        180
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьев
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорий
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьевич
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        220
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 2`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Иванов
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иван
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иванович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        200
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Сидоров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидор
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидорович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        250
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Петров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Пётр
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Петрович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        180
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьев
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорий
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьевич
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        220
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 3`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Иванов
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иван
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иванович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        200
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Сидоров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидор
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидорович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        250
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Петров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Пётр
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Петрович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        180
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьев
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорий
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьевич
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        220
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 4`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Иванов
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иван
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иванович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        200
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Сидоров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидор
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидорович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        250
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Петров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Пётр
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Петрович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        180
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьев
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорий
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьевич
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        220
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 5`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 6`] = `
<div
  className="MobileCompany"
>
  <div
    className="MobileCompany__buttonsWrap"
  >
    <input
      onClick={[Function]}
      type="button"
      value="Все"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Активный"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Заблокированные"
    />
  </div>
  <div
    className="MobileCompany__clientsWrap"
  >
    <div
      className="clientsWrap__header"
    >
      <span
        className="header__cell"
      >
        Фамилия
      </span>
      <span
        className="header__cell"
      >
        Имя
      </span>
      <span
        className="header__cell"
      >
        Отчество
      </span>
      <span
        className="header__cell"
      >
        Баланс
      </span>
      <span
        className="header__cell"
      >
        Статус
      </span>
      <span
        className="header__cell"
      >
        Редактировать
      </span>
      <span
        className="header__cell"
      >
        Удалить
      </span>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Иванов
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иван
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Иванович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        200
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Сидоров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидор
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Сидорович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        250
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Петров
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Пётр
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Петрович
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        180
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьев
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорий
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        Григорьевич
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        220
      </span>
      <span
        className="clientCell MobileClient__activeBalance"
      >
        active
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
    <div
      className="MobileClient"
    >
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientFIO clientCell"
      >
        
      </span>
      <span
        className="MobileClientBalance clientCell"
      >
        0
      </span>
      <span
        className="clientCell MobileClient__blockedBalance"
      >
        blocked
      </span>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Редактировать"
        />
      </div>
      <div
        className="clientCell"
      >
        <input
          onClick={[Function]}
          type="button"
          value="Удалить"
        />
      </div>
    </div>
  </div>
  <input
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;
